Strategic-Initiatives.md

```diff
@@ -51,4 +50,4 @@ and have the support needed.
 [rvagg]: https://github.com/rvagg
 [trott]: https://github.com/trott
 [trevnorris]: https://github.com/trevnorris
```

Comment:
I think this is the line that would be removed.

Reply:
Thanks, didn't see that!